test/controllers/entry_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class EntryControllerTest < ActionDispatch::IntegrationTest
+  test "should create entry associated with a list which is then associated with a user" do
```

Comment:
Looks like you brought in some Minitest for your testing instead of Rspec.  Do you have a preference for one or the other?  If you like/understand Minitest, I'll rewrite the rspec ones to follow it.

Reply:
Oh! I thought I was using RSpec , sorry!